index.html

```diff
@@ -233,6 +250,14 @@ <h3>
               maxlength="30"
               required
             />
+            <input
+              type="text"
+              name="name"
+              id="user_lname"
```

Comment:
- [ ] I can see you have two inputs for Full name. Kindly ensure that there should be only one input for Full name (also in the mobile version) and this should take the space of both First Name and Last Name inputs.